src/js/jsx/Scrim.jsx

```diff
@@ -28,14 +28,25 @@ define(function (require, exports, module) {
     var React = require("react"),
         Fluxxor = require("fluxxor"),
         FluxMixin = Fluxxor.FluxMixin(React),
-        StoreWatchMixin = Fluxxor.StoreWatchMixin;
+        StoreWatchMixin = Fluxxor.StoreWatchMixin,
+        classnames = require("classnames");
 
     var adapterOS = require("adapter/os");
 
-    var PolicyOverlay = require("jsx!js/jsx/tools/PolicyOverlay");
-
+    var PolicyOverlay = require("jsx!js/jsx/tools/PolicyOverlay"),
+        Droppable = require("jsx!js/jsx/shared/Droppable");
+
+    /**
+     * Indicates whether the mouse is currently within the view port of the Scrim component. It is shared between the 
+     * Scrim component and the "canDropGraphic" callback. 
+     * @private
+     * 
+     * FIXME need a better way to share information between component and Droppable settings.
+     */
+    var _isMouseInsideScrim = false;
```

Comment:
For the current drag-and-drop implementation, this seems to be the only way to share information between component and droppable [callbacks](https://github.com/adobe-photoshop/spaces-design/pull/1855/files#diff-768f12e2b3c6382be0ac86fe532542feR323), if you don't want to get store involved.